scripts/build-circt.sh

```diff
@@ -3,6 +3,20 @@ set -eu
 
 GIT_COMMIT=093cdfe482530623fea01e1d3242af93e533ba54
 
+# Get the absolute path to this script and set default build and install paths
+SCRIPT_DIR="$(dirname "$(realpath "$0")")"
+CIRCT_BUILD=${SCRIPT_DIR}/../build-circt
+CIRCT_INSTALL=${SCRIPT_DIR}/../usr
```

Comment:
Do I understand this correctly in that the installation is in the end in `.../jlm/usr` (if the below statements do not kick in)?

Reply:
Yes, the default would be to place it in jlm/usr